src/cli/utils.ts

```diff
@@ -0,0 +1,129 @@
+import fs from "fs";
+
+import globTool from "glob-promise";
+
+interface Config {
+  [key: string]: {
+    element: string;
+    css: string;
+    variants: {
+      [key: string]: {
+        [key: string]: string;
```

Comment:
Make this a VariantOption type. Then you can use it in a few places - line 15 and on line 64 too